instrumentation/kafka-clients/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.zipkin.brave</groupId>
+    <artifactId>brave-instrumentation-parent</artifactId>
+    <version>4.5.3-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>brave-instrumentation-kafka</artifactId>
+  <name>Brave Instrumentation: Kafka</name>
+
+  <properties>
+    <main.basedir>${project.basedir}/../..</main.basedir>
+    <kafka.version>0.11.0.0</kafka.version>
```

Comment:
if kafka 0.11 requires >= java 8, then you can disable retrolambda:

```xml
      <plugin>
        <groupId>net.orfjackal.retrolambda</groupId>
        <artifactId>retrolambda-maven-plugin</artifactId>
        <executions>
          <execution>
            <phase>none</phase>
          </execution>
        </executions>
      </plugin>
```

and set the following properties:
```
    <main.java.version>1.8</main.java.version>
    <main.signature.artifact>java18</main.signature.artifact>
```